src/Docfx.Dotnet/Docfx.Dotnet.csproj

```diff
@@ -34,6 +34,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.Formats.Asn1" />
```

Comment:
`System.Formats.Asn1` is transitively referenced by `Microsoft.CodeAnalysis.Workspaces.MSBuild` package.
So If this package is updated. It can removed this setting.